<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-AB1UVI3" ignored="8" name="Default test" tests="1" failures="1" timestamp="13 Jul 2021 12:26:04 GMT" time="7.014" errors="0">
  <testcase name="verifyHeaderOfUPTest" time="0.226" classname="com.jbk.Test.UserPageTest">
    <failure type="java.lang.AssertionError" message="lists don&amp;apos;t have the same size expected [8] but found [5]">
      <![CDATA[java.lang.AssertionError: lists don't have the same size expected [8] but found [5]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:556)
	at org.testng.Assert.assertEquals(Assert.java:533)
	at com.jbk.Test.UserPageTest.verifyHeaderOfUPTest(UserPageTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- verifyHeaderOfUPTest -->
  <testcase name="UserGenderTest" time="0.0" classname="com.jbk.Test.UserPageTest">
    <ignored/>
  </testcase> <!-- UserGenderTest -->
  <testcase name="addUserTest" time="0.0" classname="com.jbk.Test.UserPageTest">
    <ignored/>
  </testcase> <!-- addUserTest -->
  <testcase name="VerifyGenderTest" time="0.0" classname="com.jbk.Test.UserPageTest">
    <ignored/>
  </testcase> <!-- VerifyGenderTest -->
  <testcase name="verifyStateTest" time="0.0" classname="com.jbk.Test.UserPageTest">
    <ignored/>
  </testcase> <!-- verifyStateTest -->
  <testcase name="VerifyValidMobnoTest" time="0.0" classname="com.jbk.Test.UserPageTest">
    <ignored/>
  </testcase> <!-- VerifyValidMobnoTest -->
  <testcase name="UserPageTitleTest" time="0.0" classname="com.jbk.Test.UserPageTest">
    <ignored/>
  </testcase> <!-- UserPageTitleTest -->
  <testcase name="printUserTableTest" time="0.0" classname="com.jbk.Test.UserPageTest">
    <ignored/>
  </testcase> <!-- printUserTableTest -->
  <testcase name="ContactDetailsTest" time="0.0" classname="com.jbk.Test.UserPageTest">
    <ignored/>
  </testcase> <!-- ContactDetailsTest -->
</testsuite> <!-- Default test -->
